crates/oxc_ast/src/serialize.rs

```diff
@@ -820,6 +820,28 @@ impl ESTree for ExportAllDeclarationWithClause<'_, '_> {
     }
 }
 
+#[ast_meta]
+#[estree(
+    ts_type = "ImportOrExportKind",
+    raw_deser = "
+        const exportKind = DESER[ImportOrExportKind](POS_OFFSET.export_kind);
+        THIS.declaration?.declare ? 'type' : exportKind
```

Comment:
```suggestion
        THIS.declaration?.declare ? 'type' : THIS.exportKind
```

First I tried this. But this seems to generate invalid JS code as a result. 🤔